com.unity.multiplayer.mlapi/Runtime/Transports/NetworkTransport.cs

```diff
@@ -99,6 +100,7 @@ public TransportChannel[] MLAPI_CHANNELS
             new TransportChannel(NetworkChannel.AnimationUpdate, NetworkDelivery.ReliableSequenced),
             new TransportChannel(NetworkChannel.NavAgentState, NetworkDelivery.ReliableSequenced),
             new TransportChannel(NetworkChannel.NavAgentCorrection, NetworkDelivery.UnreliableSequenced),
+            new TransportChannel(NetworkChannel.NetworkVariable, NetworkDelivery.ReliableFragmentedSequenced),
```

Comment:
Add comment please akin to "currently, fragmentation support needed to deal with oversize packets encounterable with current pre-snapshot code"